src/chi-vue/src/components/data-table-filters/DataTableFilters.tsx

```diff
@@ -11,16 +11,23 @@ import {
 } from '@/constants/classes';
 import { getElementFilterData, updateFilterData } from './FilterUtils';
 import { DATA_TABLE_EVENTS } from '@/constants/events';
-import AdvancedFilters from './AdvancedFilters';
+import AdvancedFiltersPopOver from './AdvancedFiltersPopover';
 import DataTableToolbar from '@/components/data-table-toolbar/DataTableToolbar';
+import AdvancedFilters from './AdvancedFilters';
+import Drawer from '../drawer/drawer';
+import store, { STORE_KEY } from '@/store/index';
+import { getModule } from 'vuex-module-decorators';
 
 @Component
 export default class DataTableFilters extends Vue {
   @Prop() filtersData?: DataTableFiltersData;
+  @Prop() portal?: boolean;
 
   _filtersData?: DataTableFiltersData;
   _advancedFiltersData?: DataTableFilter[];
-  _advancedFilterComponent?: AdvancedFilters;
+  _advancedFilterComponent?: AdvancedFiltersPopOver;
+  drawerActive?: boolean = false;
+  storeModule?: any;
```

Comment:
```suggestion
  // eslint-disable-next-line
  storeModule?: any;
```

Reply:
hey @janibolkvadze it is not necessary I turned off the logic in es linter. I don't think the rule of not using `any` type be helpful in the project because it is necessary sometimes